extensions/tls-registry/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -8,3 +8,5 @@ metadata:
   unlisted: true
   config:
   - "quarkus.tls."
+  cli-plugins:
+  - "${project.groupId}:quarkus-tls-registry-cli:${project.version}"
```

Comment:
```suggestion
  - tls: "${project.groupId}:quarkus-tls-registry-cli:${project.version}"
```

Reply:
As of https://github.com/quarkusio/quarkus/pull/40580 we can prefix the maven coordinates with an alias.
In this case `tls: `.

Reply:
Awesome!

Reply:
Did the changes.

```yaml
  cli-plugins:
  - "tls: ${project.groupId}:quarkus-tls-registry-cli:${project.version}"
```